rocks-lib/src/rockspec/build/utils/mod.rs

```diff
@@ -29,8 +29,18 @@ pub fn compile_c_files(files: &Vec<PathBuf>, target_file: &str, target_dir: &Pat
     let target = lua_module_to_pathbuf(target_file, std::env::consts::DLL_SUFFIX);
     let target = target_dir.join(target);
 
-    let parent = target.parent().expect("TODO");
-    let file = target.file_name().expect("TODO");
+    let parent = target.parent().unwrap_or_else(|| {
+        panic!(
+            "Couldn't determine parent for path {}",
```

Comment:
didn't see anything like it in the code
but do you perhaps have any standards for error messages? E.g. adding a prefix to indicate where they come from?

Reply:
We'll be estabilishing a standard from error messages once me move over to the ariadne crate for everything. For now, this is fine :)